docs/alerts/scheduled-searches/create-email-alert.md

```diff
@@ -28,15 +32,91 @@ Before creating an email alert, you'll first need to [create a scheduled search]
+
+### Scheduled email
+
+Provides a summary of the search results every time a scheduled search is run. The subject of this email is **Search Results**. No matter what the results of the search, you'll receive an email.
+
+Email notifications are sent to the recipient email address(es) based on the notification parameters. For queries returning high numbers of messages or group results, the email notification contains a representative sample of the first 25 messages or groups. **Most Recent Results** also **displays** the number of remaining results available, which you can view in Sumo Logic with a click.
+
+### Alert email
+
+Set up Alert Conditions if you'd rather just receive an alert email when certain conditions are met. This means you'd get an automated alert when or if parameters you set are triggered, based on the results of the scheduled search. For example, you could set up an alert if a certain number of users visit the free trial URL of your website. You can [run a search from this email](#run-a-search-from-an-alert-email) when you receive it.
+
+You can be very specific with the alert condition — you can even set an exact number of results that triggers the email. Results can either be the number of log messages *OR* the number of aggregates returned by the saved search. If your saved search returns log messages, then the alert will use the number of messages you specify. If your query produces aggregate results, the alert will use the number of aggregates (or groups).
+
+#### Run a search from an alert email
+
+You can run a search directly from an Alert Email. When you receive an email created by an Email Alert in a Scheduled Search, only the first 25 results are displayed. (We do not want to flood your email with hundreds of thousands of search results.)
+
+Do either of the following:
+* To make changes to the search query before you run it again, click the saved search title link, next to **Saved Search**. This will open the query in the Sumo Logic search page. 
+* To see all the results of the search, under **Message Distribution**, click the **View results in Sumo Logic** link in the email. Or under **Aggregation**, click "**here**". Sumo Logic will recreate the search exactly matching the query and time parameters of the original scheduled search.
+
+![Search from email](/img/alerts/search_from_email_new.png)
+
+:::note
+If you're a new user and someone has forwarded you an alert email, the links to the search will not work until you've completed your setup process.
+:::
+
+
+### Real-time alerts
+
+:::note
+Not available to Sumo Logic Free accounts.
+:::
+
+Real Time Alerts continuously monitor your Sumo Logic deployment, and return alert emails whenever conditions are met. You can learn more in [Create a Real Time Alert](create-real-time-alert.md).
+
+Scheduled searches run according to the time zone of an individual's computer and browser, not according to the time zone of logs.
```

Comment:
Should you even include the "Real-time alerts" section if the feature is [deprecated](https://help.sumologic.com/docs/alerts/scheduled-searches/deprecation/)?

Reply:
Good call. For consistency with [this doc](https://help.sumologic.com/docs/alerts/scheduled-searches/create-real-time-alert/), I'll add the deprecation warning for now.